js/config.js

```diff
@@ -0,0 +1,3 @@
+// Contains "global" variables. 
+
+var startLevelKey = "L10";
```

Comment:
For a global variable intended to be a constant, I find naming the var in all caps to be helpful: `START_LEVEL_KEY`.